### ISSUE TITLE
what is the logits: return value of net = Unet

### ISSUE THREAD

**Issue by hilmicanguven** [milesial/Pytorch-UNet#248]
sir, can you explain what is this logits variable (return value of Unet class)? and what values are return?  @milesial

**Comment by milesial** [comment]
The network returns 2D maps of logits. https://en.wikipedia.org/wiki/Logit
